### DISCUSSION TITLE
[CSV Import] How to update a "many-to-many" attribute?

### DISCUSSION THREAD

**Discussion by pento-thal** [pimcore/pimcore#10186] [Q&A / Support]
Hi, I trying to update a **many-to-many** attribute by using a csv file.

As you can see the file is composed by one column for each item (in this case objects) I would like to add to my many-to-many attribute.
![image](https://user-images.githubusercontent.com/83424908/131142736-e90bd6c8-9bce-48ed-9bf2-4aeaca01c4b5.png)
The result I would like is this one:
![image](https://user-images.githubusercontent.com/83424908/131143673-10d915ec-f77e-49c4-baf5-a096896020af.png)
So, I'm trying to manage this type of update through a csv upload. So I made this column configuration:
![image](https://user-images.githubusercontent.com/83424908/131143994-831b788f-23e0-4c4a-963e-f621b9120f40.png)
The update is successful for the one-to-many attribute, but nothing happens for the many-to-many ones.

Does anybody knows how should I configure my upload?
Thanks.

**Comment by pento-thal** [comment]
Can anyone help me?

**Comment by aweichler** [comment]
Hello, I think you should better use [pimcore/data-importer](https://github.com/pimcore/data-importer). You can do this with the data-importer, but only if you have a fixed number of columns for the many to many field. Otherwise, it would be better to have a column where the values are separated by a character (example: |). In the data importer you can separate the value with the character, treat it like an array, load data objects from the array and put the object into a field.
